.tekton/sandboxed-containers-operator-devel-pull-request.yaml

```diff
@@ -0,0 +1,401 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift/sandboxed-containers-operator?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "devel"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: sandboxed-containers-operator
+    appstudio.openshift.io/component: sandboxed-containers-operator-devel
```

Comment:
Is this suffix "-devel" something we can control? or it was automatic? Currently our component name is "osc-operator"